docs/local_setup_sans_docker.md

```diff
@@ -46,6 +82,8 @@ Now, you should be about ready to get started. This command in the top folder of
 bin/dev
 ```
 
+If you get an error related to `puma` and/or trying to connect to an occupied port, it could be because your OS is using port 5000 for Airplay. Follow [these instructions](https://medium.com/pythonistas/port-5000-already-in-use-macos-monterey-issue-d86b02edd36c) to turn off Airplay and liberate that port.
```

Comment:
🔧 Sorry for ✂️ my own suggestion, but realized this can be addressed by a far less intrusive fix, by updating the port settings!